### PR TITLE
Added the test cases for Meta tag api

### PR DESCRIPTION
**Pull request by sinchanagoudar**

### CODE REVIEW

File: cypress/e2e/MetaTagsAPI/documentation.cy.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    it('5. Test to check "Meta Tags API Reference" title is present', () => {
+        cy.checkTextVisibility('Meta Tags API Reference');
+    });
+
+    it('6. Test to check "Just Getting Started?" title is present', () => {
+        cy.checkTextVisibility('Just Getting Started?');
+    });
+
+    it('7. Test to check "development quickstart guide" link visibility and click behavior', () => {
+        cy.componentVisiblityCheck('a[href="/docs/article/getting-started"]', 'development quickstart guide.');
```
Comment (AngadCel) [inline cypress/e2e/MetaTagsAPI/documentation.cy.js]: assign on variable and use it the same

File: cypress/e2e/MetaTagsAPI/documentation.cy.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        cy.contains('span', 'Documentation').should('be.visible').click();
+    });
+
+    it('8. Test to check "Authentication" link visibility and content', () => {
+        cy.componentVisiblityCheck('a[href="#authentication"]', 'Authentication');
+    });
+
+    it('9. Test to click on "Authentication" hyperlink and check "Authentication" title is present', () => {
+        cy.contains('a', 'Authentication').click();
+        cy.componentVisiblityCheck('h1, h2, h3, h4', 'Authentication');
```
Comment (AngadCel) [inline cypress/e2e/MetaTagsAPI/documentation.cy.js]: change it and same for all

File: cypress/e2e/MetaTagsAPI/home.cy.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    it('4. Test to check "Live Demo" button is present', () => {
+        cy.componentVisiblityCheck('.show-code', 'Live Demo');
+    });
+
+    it('5. Test to check code example  is present', () => {
+        cy.componentVisiblityCheck('.code-response');
+    });
+
+    it('6. Test to click "check the documentation" hyperlink is working', () => {
+        cy.get('p > a').click()
```
Comment (AngadCel) [inline cypress/e2e/MetaTagsAPI/home.cy.js]: change it to the last suggested one

File: cypress/e2e/MetaTagsAPI/documentation.cy.js (hunk 1 above)
Comment (sinchanagoudar) [reply cypress/e2e/MetaTagsAPI/documentation.cy.js]: done

File: cypress/e2e/MetaTagsAPI/home.cy.js (hunk 3 above)
Comment (sinchanagoudar) [reply cypress/e2e/MetaTagsAPI/home.cy.js]: Done

File: cypress/e2e/MetaTagsAPI/documentation.cy.js (hunk 2 above)
Comment (sinchanagoudar) [reply cypress/e2e/MetaTagsAPI/documentation.cy.js]: Done

**Pull request merged**